index.html

```diff
@@ -284,28 +284,28 @@ <h2>Nextcloud 16<a class="headerlink" href="#nextcloud-16" title="Permalink to t
 								<h1>Nextcloud Desktop Client<a class="headerlink" href="#desktop" title="Permalink to this headline">¶</a></h1>
 								<p>Once you have a Nextcloud Server running, you can connect to it with various <a class="reference external" href="https://nextcloud.com/install/#install-clients">clients like our mobile and desktop client.</a>
 								Find documentation for the desktop client below.</p>
-								<div class="section" id="desktop-30">
-									<h2>Desktop 3.0<a class="headerlink" href="#desktop-30" title="Permalink to this headline">¶</a></h2>
+								<div class="section" id="desktop-31">
+									<h2>Desktop 3.1<a class="headerlink" href="#desktop-31" title="Permalink to this headline">¶</a></h2>
 									<p>This documents the <em>latest</em> version of the Nextcloud desktop client.</p>
 									<ul class="simple">
-										<li><a class="reference external" href="https://docs.nextcloud.com/desktop/3.0">Manual</a>
+										<li><a class="reference external" href="https://docs.nextcloud.com/desktop/3.1">Manual</a>
```

Comment:
Overall LGTM, but I get a 404 on that one.

Reply:
fixed